editors/code/src/commands/server_version.ts

```diff
@@ -0,0 +1,10 @@
+import * as vscode from 'vscode';
+import { ServerVersion } from '../installation/server';
+import { Cmd } from '../ctx';
+
+export function serverVersion(): Cmd {
+    return () => {
+        vscode.window.showInformationMessage('rust-analyzer version : ' + ServerVersion);
```

Comment:
Yeah, rather than relying on ServerVersion being present, I'd rather just call `ensureServerBinary` and then `--version` for re resulting binary. It might actually throw any errors associated with downloading the binary again, but this is actually something we do want in this command!

Reply:
It will also propose you to install the binary on the road)))

Reply:
Argeed and fixed.